templates/org/menu.tmpl

```diff
@@ -40,8 +40,9 @@
 			</a>
 			{{end}}
 			{{if .IsOrganizationOwner}}
+			<span class="item-flex-space"></span>
```

Comment:
Would name it `overflow-menu-spacer`

Reply:
Also consisten with MacOS, but let's name it `.flexible-space` then, maybe it's useful outside of overflow-menu at some point.

<img width="107" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/0b08d329-c769-47bc-8593-de671070abc1">

Reply:
Guess we don't have to bikeshed this name.